02-peer-protocol.md

```diff
@@ -316,14 +339,21 @@ The sender:
 avoid double-spending of the funding transaction.
   - MUST set `channel_reserve_satoshis` greater than or equal to `dust_limit_satoshis` from the `open_channel` message.
   - MUST set `dust_limit_satoshis` less than or equal to `channel_reserve_satoshis` from the `open_channel` message.
+  - if it sets `channel_type`:
+    - MUST set it to one of the `channel_types` from `open_channel`
+    - SHOULD set it to its preferred `channel_type`, or the first in `channel_types` if there is more than one equal preference.
```

Comment:
Nit: the last part derives from the rest and seems a bit too verbose, in fact the whole line could be removed imo

```suggestion
    - SHOULD set it to its preferred `channel_type`.
```

Reply:
Yeah, we got rid of this "array of channel_types" anyway, so it's even less verbose now :)